.github/workflows/test.yml

```
@@ -9,14 +9,16 @@ on:
       - 'docs/**'
 
 jobs:
-  pytest:
+  ctest:
```

[consider] Then we don't have to keep changing the "required checks" in the branch protection rules in the future if we change the test runner again. 
```suggestion
  test:
```